wrappers/python/examples/wnt/README.md

```diff
@@ -9,7 +9,7 @@ as the examples about the message flow and content.
 ## Requirements
```

Comment:
As a general comment, should the examples changes be done already in new repo?

Reply:
Examples are in different repo. Only protos are here.